modular-docs-manual/content/topics/using_text_fragments.adoc

```diff
@@ -0,0 +1,19 @@
+[id="using_text_fragments_{context}"]
+
+= Text Fragments
+
+A text fragment is a small section of text that is stored in an Asciidoc file. Text fragments contain content that is resused in multiple modules or assemblies, for example:
+
+* A step or series of steps in a procedure
+* A disclaimer, for example, for technology preview or beta releases
+
+NOTE: A text fragment is not a module. The Asciidoc file that contains the text fragment cannot contain any other information. A module cannot contain an include statement for another module.
+
+.Procedure
+. Create the text fragment Asciidoc file and store it in your fragments folder.
```

Comment:
> store it in your fragments folder

I understand that this is good practice, but do we need to call it out. In the end, it's up to the writer (or the writing team) how they wish to store their `.adoc` files.

```suggestion
. Create the text fragment AsciiDoc file and store it in your fragments folder.
```

Reply:
I imagine that readers will appreciate the suggested directory name and many will adopt it for the benefits of consistency.  Rule breakers do whatever they want anyway.

Reply:
I made it a suggestion and put it in a note.